vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/ResourceCheckSteps.java

```diff
@@ -96,37 +98,43 @@ public void init()
      *     3. If element doesn't contain href or src attribute fail assertion will be recorded
      * @param cssSelector to locate resources
      * @param html to validate
-     * @throws InterruptedException when a thread is interrupted
-     * @throws ExecutionException  when exception thrown before result get
+     * @throws Exception when exception occurred
      */
     @Then("all resources by selector `$cssSelector` from $html are valid")
-    public void checkResources(String cssSelector, String html) throws InterruptedException, ExecutionException
+    public void checkResources(String cssSelector, String html) throws Exception
```

Comment:
it's better to keep all exception instead of just `Exception`

Reply:
> it's better to keep all exception instead of just `Exception`

Do you mean just left subclass exceptions (`InterruptedException`, `ExecutionException`) in method signature and javadocs?